v2/tools/asoctl/internal/crd/cleaner.go

```diff
@@ -78,7 +83,8 @@ func (c *Cleaner) Run(ctx context.Context) error {
 
 		// If the slice was not updated, there is no version to deprecate.
 		if len(newStoredVersions) == len(crd.Status.StoredVersions) {
-			klog.Infof("Nothing to update for %q\n", crd.Name)
+			c.log.Info("Nothing to update",
```

Comment:
minor: kinda weird linebreaks here.

Consider:
```
c.log.Info(
    "Nothing to update",
    "crd-name", crd.Name)
```

or just putting everything on one line since it's pretty short.

Reply:
Applies below too

Reply:
Deliberately put each name/value pair on a separate line to make them easier to visually parse.

Reply:
That's fine, but you should put a newline after the `Info(`, otherwise everything ends up over-tabbed, right?

Reply:
Agreed. Updated.